README.md

```diff
@@ -156,9 +156,9 @@ If you like this project just give it a star.
 
 <!-- ACKNOWLEDGEMENTS -->
 
-## 🙏 Acknowledgments <a name="acknowledgements"></a>
+## 🙏 Acknowledgments <a name="acknowledgement"></a>
 
-I would like to thank Microverse.
+- Thank you  <a href="#"> Cindy Shin</a> for designing the teplate!
 
```

Comment:
- [ ] You have done a great job 💪of adding the acknowledgment👍 section as suggested by the previous reviewer🙄. However, you should add a link behind 'Cindy Shin' like this: [Cindy Shin](https://www.behance.net/adagio07). This will allow users of your project to follow the link to the original design.

Reply:
hello sir you mean I should put like that?
The Original design idea is by [Cindy Shin in Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) Thanks to **Cindy Shin** for Creating such a beautiful design.